File: app/src/main/java/me/eagzzycsl/darkroom/ui/PickPagerAdapter.java
package me.eagzzycsl.darkroom.ui;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;

public class PickPagerAdapter extends FragmentPagerAdapter {
    private String[] title = {
            "用户应用",
            "系统应用",
    };
    private Fragment[] fragments = {};

    public PickPagerAdapter(FragmentManager fm, FragmentOnDeviceApp[] fragments) {
        super(fm);
        this.fragments = fragments;

    }

    @Override
    public Fragment getItem(int position) {
        return fragments[position];
    }

    @Override
    public CharSequence getPageTitle(int position) {
        return title[position];
    }

    @Override
    public int getCount() {
        return fragments.length;
    }
}

File: app/src/main/java/me/eagzzycsl/darkroom/ui/PickPagerAdapter.kt
package me.eagzzycsl.darkroom.ui

import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentPagerAdapter

class PickPagerAdapter(fm: FragmentManager, fragments: Array<FragmentOnDeviceApp>) : FragmentPagerAdapter(fm) {
    private val title = arrayOf("用户应用", "系统应用")
    private var fragments = arrayOf<FragmentOnDeviceApp>()

    init {
        this.fragments = fragments

    }

    override fun getItem(position: Int): Fragment {
        return fragments[position]
    }

    override fun getPageTitle(position: Int): CharSequence {
        return title[position]
    }

    override fun getCount(): Int {
        return fragments.size
    }
}
